### ISSUE TITLE
Possible crash issue

### ISSUE THREAD

**Issue by llandyw** [valinet/ExplorerPatcher#87]
I don't know if it's the dll or something in the latest Windows 11 update, but:

My additional toolbars disappeared, but I can add them back in, and the menu seems unaltered.
However, Explorer has crashed twice when doing a search (they brought back search as you type).

As I said, I don't know where the issue is, but thought you should be aware of it.

Andy

**Comment by valinet** [comment]
Can you provide a more detailed description? 

> they brought back search as you type
What's this? Where?

**Comment by llandyw** [comment]
Ok, as far as the search as you type, it seems to have quit again.  Explore search is what I was referring to.  Folder window, search textbox.  Used to be that when you start typing your search term, it would begin searching automatically.  It worked once, then quit.  This was right after the update I received today.

The taskbar toolbars are what disappeared after the Windows update.  I put them back, then updated the dxgi DLL and they disappeared again.  I was only able to replace 2 instances as the third one in the directory with CBS in its name is the one that I haven't been able to replace.  Says it's in use, with Explorer shut down.  The right click menu on the toolbar is the one I was referencing.  It didn't seem to get affected by the Windows update.

Since the search quit, I can't get a screenshot.  I really wish MS hadn't made it so you have to hit enter to begin a search!

Also, after replacing the dxgi, no explorer crashes, at least not yet.  But I have restarted multiple times since the update.

Andy

**Comment by llandyw** [comment]
I am on 22000.258 using the 22000.258.26.2 dxgi.dll.

I did not have to turn off Focus Assist, it was already off (I've never changed it since the upgrade from W10 to W11).

I also haven't seen a notice about downloading symbols, but the classic taskbar is there.

I already had the previous build of dxgi.dll in the 3 places mentioned in the setup instructions at the time.
When I downloaded the .256.26.2 version, I saved it into the Windows folder (renamed), opened a cmd window, shut down Explorer, deleted the old version and renamed the new version to its rightful name.  Started Explorer and reset up my toolbars.  I keep my toolbars in their own folder, do there's 10 toolbars along with pinned items on the taskbar.

Those had disappeared when Windows 11 was updated to .258.  I had missed the part about not needing to update dxgi.dll in the two other folders, so with Explorer off, I updated one of them, but the other would not update as it was in use even though Explorer was shut down.

Should I remove or replace either or both of the dxgi.dll files from the 2 extra folders?

I have however run across another issue.

I changed the settings to disable the modern search bar, open Start to All apps by default and Enable Console.

The console window comes up, but if I close it, it restarts Explorer again and comes right back up.
Is this how it's supposed to work?  I would have thought that closing it should just let it close without doing anything else since it seems to only be there to show what's being done when settings are changed as it doesn't accept input of any kind that I've been able to see.

Andy
